spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/jni-config.json

```diff
@@ -0,0 +1,2 @@
+[
```

Comment:
Don't include empty file, same problem need to be done below.

Reply:
Done